Assembler.Base/InitialFrameHandler.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+
+namespace Assembler.Base
+{
+    public class InitialFrameHandler<TFrame, TMessage> : BaseFrameHandler<TFrame, TMessage>
```

Comment:
Use messageInAssembly as varible name instead of message, thats not clear enough

Reply:
f